packages/trpc/server/routers/viewer/organizations/_router.tsx

```diff
@@ -98,4 +100,19 @@ export const viewerOrganizationsRouter = router({
       input,
     });
   }),
+  getBrand: authedProcedure.input(ZGetBrandSchema).query(async ({ ctx, input }) => {
```

Comment:
Adding the trpc router and endpoint to get org branding properties for the provider